### ISSUE TITLE
Accuracy using pretrained weights is 0.69 

### ISSUE THREAD

**Issue by zhanxw** [IBM/molformer#20]
The frozen_embeddings_classification.ipynb reported that the accuracy is 0.86.
I rerun this notebook on two servers, and both have accuracy around 0.69. (notebook attached).

Has anyone else noticed a similar decrease in accuracy?

[replicate.ipynb.zip](https://github.com/IBM/molformer/files/14398110/replicate.ipynb.zip)

**Comment by mtinti** [comment]
if I use:
test_df = pd.read_csv('../../data/bace/test.csv')
insted of:
test_df = pd.read_csv('../../data/bace/test.csv').sample(frac=0.1)

I get ROC 0.8374908290535583

**Comment by xdn-github** [comment]
Yey, I got 0.69 too, 

What's more, this pre-train model was trained on the 10%Zinc+10%PubChem as described in README(fig shonw below), which is NOT the best model trained on 1.1B datasets. 
![image](https://github.com/user-attachments/assets/46c0f864-3058-499a-887a-4e57e62bfd33)

 BUT, even for this pre-trained model, it shoud have 86.6 on BACE test. Scores from the [preprint version assay](https://arxiv.org/abs/2106.09553) as shown below
![image](https://github.com/user-attachments/assets/bb45459b-2c60-4cfc-9e76-42a5334f5f60)

I don't know which step I got wrong..
